### ISSUE TITLE
Content not assigned to workflow can fail to push publish

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#22087]
If you include a contentlet that is not in a workflow step in a bundle, it fails to publish.
Easy to reproduce.  Take a contentlet and run the "reset workflow" actionlet on it, which will leave it in the "Not Assigned State".  Try to push it.

![Screen Shot 2022-04-26 at 11 31 54 AM](https://user-images.githubusercontent.com/934364/165337652-2c7ebc79-74a5-4abd-9933-97ed8533bdfa.png)

![Screen Shot 2022-04-26 at 11 32 35 AM](https://user-images.githubusercontent.com/934364/165337664-7d76c75b-608f-404e-a60f-e72d8b52b4da.png)

**Comment by fabrizzio-dotCMS** [comment]
The exception comes from an NPE on the receiver that prevents completing the transaction

```
java.lang.NullPointerException: null
[09/05/22 21:52:15:118 UTC] ERROR receiver.BundlePublisher: Error publishing bundle with ID '01G2NCCCREY9A2PC6DH05Z7D40.tar.gz': An error occurred when processing Workflow Task in '/data/shared/assets/bundles/01G2NCCCREY9A2PC6DH05Z7D40/live/demo.dotcms.com/1/00f9c5ac0cb5c71a194170a68d879ca6.contentworkflow.xml' with ID '2_minutes.jpeg': 967c3f43-0b7e-4f67-bcda-55dc38f4eebe
com.dotcms.publishing.DotPublishingException: An error occurred when processing Workflow Task in '/data/shared/assets/bundles/01G2NCCCREY9A2PC6DH05Z7D40/live/demo.dotcms.com/1/00f9c5ac0cb5c71a194170a68d879ca6.contentworkflow.xml' with ID '2_minutes.jpeg': 967c3f43-0b7e-4f67-bcda-55dc38f4eebe
at com.dotcms.enterprise.publishing.remote.handler.ContentWorkflowHandler.a(SourceFile:188) ~[ee-22.05.jar:?]
at com.dotcms.enterprise.publishing.remote.handler.ContentWorkflowHandler.handle(SourceFile:94) ~[ee-22.05.jar:?]
at com.dotcms.publisher.receiver.BundlePublisher.process(BundlePublisher.java:220) ~[dotcms_22.05_999999.jar:?]
at com.dotcms.rest.PushPublisherJob.processBundle(PushPublisherJob.java:40) ~[dotcms_22.05_999999.jar:?]
at com.dotcms.rest.PushPublisherJob.run(PushPublisherJob.java:65) ~[dotcms_22.05_999999.jar:?]
at com.dotmarketing.quartz.DotJob.execute(DotJob.java:42) ~[dotcms_22.05_999999.jar:?]
at org.quartz.core.JobRunShell.run(JobRunShell.java:223) ~[dot.quartz-all-1.8.6_2.jar:?]
at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:549) ~[dot.quartz-all-1.8.6_2.jar:?]
Caused by: java.lang.NullPointerException
```

**Comment by nollymar** [comment]
PR EE: https://github.com/dotCMS/enterprise/pull/873

**Comment by nollymar** [comment]
PR Core: #22191

**Comment by nollymar** [comment]
**Internal QA:** 

Pushing a contentlet without a workflow assigned is working now but in the receiver the workflow assigned is New. Is it ok @wezell ? 

_Sender:_

<img width="840" alt="Screen Shot 2022-05-20 at 5 22 07 PM" src="https://user-images.githubusercontent.com/8741395/169619871-38f29335-c0ec-4576-adf5-9fc03d2b836b.png">

_Receiver:_

![Screen Shot 2022-05-20 at 5 21 49 PM](https://user-images.githubusercontent.com/8741395/169619939-a6beac31-d205-4ea4-8370-bfff10823a0b.png)

**Comment by wezell** [comment]
Yes. Expected.

**Comment by nollymar** [comment]
Great! I'm passing the internal QA then. Thanks :)

**Comment by bryanboza** [comment]
Fixed, tested on release-22.06 // Docker // FF
